app/uk/gov/hmrc/accessibilitystatementfrontend/models/AccessibilityStatement.scala

```diff
@@ -26,6 +26,7 @@ case class AccessibilityStatement(
   serviceDescription: String,
   serviceDomain: String,
   serviceUrl: String,
+  serviceTemplateOverrideType: Option[ServiceTemplateOverrideType] = None,
```

Comment:
@kyle-bowden @JoPintoPaul seeing it in use, I wonder if we should change this to "statementTemplate" and make it non-optional

Reply:
@oscarduignan 
I guess the only problem I can see is if it's non-optional, we would then need to add this property to all existing service YAML files.

Reply:
@kyle-bowden could we rename ServiceTemplateOverrideType to TemplateType or StatementTemplateType - it feels like the config is to configure and override, but that override is Some(templateType). Everything else looks good to me 🙌

Reply:
🙌